src/factory/index.js

```diff
@@ -0,0 +1,35 @@
+type DefaultCreatorsFor<T: Object> = $ObjMap<T, <V>(V) => (() => V)>;
```

Comment:
Are you intentionally omitting the `@flow` pragma here?

Reply:
Ooooops.

Reply:
Fixed in master. 🙊